jasmine/spec/inverted-index-test.js

```diff
@@ -1,53 +1,67 @@
-/* Setups goes here */
-const sampleBookData = [
+/* Sample Test Data */
```

Comment:
You should use proper doc string comments here and all recurring improper doc string comments throughout the rest of your code. 
Visit [usejsdoc](usejsdoc.org) for more knowledge on that

**Example** 
`/** Comment goes */`